2023.04-4week/hyunwlee/[JS] BOJ.1018.체스판 다시 칠하기/체스판 다시 칠하기.js

```diff
@@ -0,0 +1,42 @@
+const readline = require("readline");
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const input = [];
+rl.on("line", function (line) {
+  input.push(line);
+}).on("close", function () {
+  solution();
+  process.exit();
+});
+
+function solution() {
+  const [N, M] = input[0].split(" ").map((item) => +item);
+  const board = Array(N).fill(Array(M).fill(null));
+  for (let i = 1; i < input.length; ++i) board[i - 1] = input[i].split("");
```

Comment:
```suggestion
  const [N, M] = input[0].split(" ").map(Number);
  const board = Array(N).fill(Array(M).fill(null));
  for (let i = 1; i < input.length; ++i) {
      board[i - 1] = input[i].split("");
   }
```
- for문에서 한 줄 표현식인가용? 한 줄 표현식이라면 자스 컨벤션에 맞지 않아서 중괄호를 남겨주는게 좋다고 합니다! 한 줄 표현식은 if문에서만 해당되는 부분이라고 하네👍🏻
- 그리고 숫자로 형변환 하는 부분은 그냥 map(Number)로 써도 무방합니다!
- 마지막으로 변수랑 for문 등 사이를 띄워주는게 보기 편할 것 같습니다.

Reply:
한줄 표현식이 자스 컨벤션에 맞지 않아서 중괄호를 남겨주는게 좋다는 멘션 저도 잘보고 갑니다 ! 😮